### PR TITLE
sci-libs/galib: Fix ISO C++17 does not allow register storage class

### PR DESCRIPTION
**Pull request by listout**
Closes: https://bugs.gentoo.org/898276

### CODE REVIEW

**Comment by listout** [comment]
Upstream seems to be dead, http://lancet.mit.edu/ga/dist/

File: sci-libs/galib/galib-2.4.7-r2.ebuild [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+	"${FILESDIR}/${PN}-2.4.7-fix-c++14.patch"
+	"${FILESDIR}/${PN}-2.4.7-Wformat-security.patch"
+)
+
+src_prepare() {
+	default
+	sed -e "s:/include:${EPREFIX}/usr/include:" \
+		-e "s:/lib:${EPREFIX}/usr/$(get_libdir):" \
+		-i makevars || die
+	find ga/ -type f -exec sed -i -e "s/register int/int/g" {} \; || die
```
Comment (thesamesam) [inline sci-libs/galib/galib-2.4.7-r2.ebuild:32]: Did you see my comments on a previous PR about sed? I'm really not a fan of it for stuff like this.

(If you can't find it, let me know,a nd I'll dig it up.)

**Comment by listout** [comment]
Based on talk over at IRC, I'm moving from the sed approach. That should also fix the ci failure.

**Comment by SoapGentoo** [comment]
@listout thanks, I've cleaned up the ebuild a bit and put all patches in the tarball. In future, just put patches in FILESDIR, even if they're large, since we have to repackage them anyways.

**Comment by listout** [comment]
@SoapGentoo Thank you very much. I'll keep this mind when hitting larger patches later on.

**Pull request closed without merging**